modules/cluster-logging-collector-log-forward-project.adoc

```diff
@@ -21,65 +21,66 @@ To configure forwarding application logs from a project, you must create a `Clus
+     outputRefs: #<9>
      - fluentd-server-insecure
      labels:
-       project: "my-project" <12>
-   - name: forward-to-fluentd-secure <13>
-     inputRefs:
-     - application <14>
+       project: "my-project" #<10>
+   - inputRefs: #<11>
+     - application #<12>
      - audit
      - infrastructure
+     name: forward-to-fluentd-secure 
      outputRefs:
      - fluentd-server-secure
      - default
      labels:
        clusterId: "C1234"
+  serviceAccount:
+    name: <service_account_name>
 ----
-<1> The name of the `ClusterLogForwarder` CR must be `instance`.
-<2> The namespace for the `ClusterLogForwarder` CR must be `openshift-logging`.
-<3> The name of the output.
-<4> The output type: `elasticsearch`, `fluentdForward`, `syslog`, or `kafka`.
-<5> The URL and port of the external log aggregator as a valid absolute URL. If the cluster-wide proxy using the CIDR annotation is enabled, the output must be a server name or FQDN, not an IP address.
-<6> If using a `tls` prefix, you must specify the name of the secret required by the endpoint for TLS communication. The secret must exist in the `openshift-logging` project and have *tls.crt*, *tls.key*, and *ca-bundle.crt* keys that each point to the certificates they represent.
-<7> The configuration for an input to filter application logs from the specified projects.
-<8> If no namespace is specified, logs are collected from all namespaces.
-<9> The pipeline configuration directs logs from a named input to a named output. In this example, a pipeline named `forward-to-fluentd-insecure` forwards logs from an input named `my-app-logs` to an output named `fluentd-server-insecure`.
-<10> A list of inputs.
-<11> The name of the output to use.
-<12> Optional: String. One or more labels to add to the logs.
-<13> Configuration for a pipeline to send logs to other log aggregators.
+<1> The name of the output.
+<2> The output type: `elasticsearch`, `fluentdForward`, `yslog`, or `kafka`.
+<3> The URL and port of the external log aggregator as a valid absolute URL. If the cluster-wide proxy using the CIDR annotation is enabled, the output must be a server name or FQDN, not an IP address.
+<4> If using a `tls` prefix, you must specify the name of the secret required by the endpoint for TLS communication. The secret must exist in the `openshift-logging` project and have *tls.crt*, *tls.key*, and *ca-bundle.crt* keys that each point to the certificates they represent.
+<5> The configuration for an input to filter application logs from the specified projects.
+<6> If no namespace is specified, logs are collected from all namespaces.
+<7> A list of inputs.
+<8> The pipeline configuration directs logs from a named input to a named output. In this example, a pipeline named `forward-to-fluentd-insecure` forwards logs from an input named `my-app-logs` to an output named `fluentd-server-insecure`.
+<9> The name of the output to use.
+<10> Optional: String. One or more labels to add to the logs.
```

Comment:
This is not correct.  This can only be done with an "openshift label" filter.  let's talk on how to use 'oc explain' so you can better understand the api